code/game/objects/effects/spawners/bombspawner.dm

```diff
@@ -1,36 +1,25 @@
-#define CELSIUS_TO_KELVIN(T_K)	((T_K) + T0C)
-
-#define OPTIMAL_TEMP_K_PLA_BURN_SCALE(PRESSURE_P,PRESSURE_O,TEMP_O)	(((PRESSURE_P) * GLOB.meta_gas_info[/datum/gas/plasma][META_GAS_SPECIFIC_HEAT]) / (((PRESSURE_P) * GLOB.meta_gas_info[/datum/gas/plasma][META_GAS_SPECIFIC_HEAT] + (PRESSURE_O) * GLOB.meta_gas_info[/datum/gas/oxygen][META_GAS_SPECIFIC_HEAT]) / PLASMA_UPPER_TEMPERATURE - (PRESSURE_O) * GLOB.meta_gas_info[/datum/gas/oxygen][META_GAS_SPECIFIC_HEAT] / CELSIUS_TO_KELVIN(TEMP_O)))
-#define OPTIMAL_TEMP_K_PLA_BURN_RATIO(PRESSURE_P,PRESSURE_O,TEMP_O)	(CELSIUS_TO_KELVIN(TEMP_O) * PLASMA_OXYGEN_FULLBURN * (PRESSURE_P) / (PRESSURE_O))
-
 /obj/effect/spawner/newbomb
 	name = "bomb"
 	icon = 'icons/hud/screen_gen.dmi'
 	icon_state = "x"
 	var/temp_p = 1500
 	var/temp_o = 1000	// tank temperatures
 	var/pressure_p = 10 * ONE_ATMOSPHERE
-	var/pressure_o = 10 * ONE_ATMOSPHERE	//tank pressures
 	var/assembly_type
 
 /obj/effect/spawner/newbomb/Initialize()
 	. = ..()
 	var/obj/item/transfer_valve/V = new(src.loc)
 	var/obj/item/tank/internals/plasma/PT = new(V)
-	var/obj/item/tank/internals/oxygen/OT = new(V)
-
-	PT.air_contents.assert_gas(/datum/gas/plasma)
-	PT.air_contents.gases[/datum/gas/plasma][MOLES] = pressure_p*PT.volume/(R_IDEAL_GAS_EQUATION*CELSIUS_TO_KELVIN(temp_p))
-	PT.air_contents.temperature = CELSIUS_TO_KELVIN(temp_p)
 
-	OT.air_contents.assert_gas(/datum/gas/oxygen)
-	OT.air_contents.gases[/datum/gas/oxygen][MOLES] = pressure_o*OT.volume/(R_IDEAL_GAS_EQUATION*CELSIUS_TO_KELVIN(temp_o))
-	OT.air_contents.temperature = CELSIUS_TO_KELVIN(temp_o)
-
-	V.tank_one = PT
-	V.tank_two = OT
+	PT.air_contents.assert_gas(/datum/gas/tritium)
+	PT.air_contents.assert_gas(/datum/gas/oxygen)
+	PT.air_contents.temperature = T20C
+	PT.air_contents.gases[/datum/gas/tritium][MOLES] = 2
+	PT.air_contents.gases[/datum/gas/oxygen][MOLES] = 3
```

Comment:
does this maxcap currently?

Reply:
Havent tested it yet, but 2*2.8e6 is 5.6e6, which is around 37 range if the math checks out. So yes it should.